### PR TITLE
Nu 1046/remove stylus toolbar components folder

### PR DESCRIPTION
**Pull request by bohdanprog**
## Checklist before merge
- [ ] Related issue ID is placed at the beginning of PR title in \[brackets\] (can be GH issue or Nu Jira issue)
- [ ] Code is cleaned from temporary changes and commented out lines
- [ ] Code formatting checked
- [ ] Parts of the code that are not easy to understand are documented in the code
- [ ] Changes are covered by automated tests
- [ ] Showcase in dev-application.conf added to demonstrate the feature
- [ ] Documentation added or updated
- [ ] Added entry in _Changelog.md_ describing the change from the perspective of a public distribution user
- [ ] Added _MigrationGuide.md_ entry in the appropriate subcategory if introducing a breaking change
- [ ] Verify that PR will be squashed during merge

### CODE REVIEW

File: designer/client/src/components/history/HistoryItem.tsx [hunk 1]
```diff
@@ -42,17 +29,15 @@ const HDate = ({ date }: { date: string }) => (
 export function HistoryItem({ onClick, version, type, isLatest, isDeployed }: HistoryItemProps): JSX.Element {
     const { t } = useTranslation();
     const { user, createDate, processVersionId, actions } = version;
-    const className = useMemo(() => mapVersionToClassName(type), [type]);
 
-    const flex = css({ display: "flex", justifyContent: "space-between", alignItems: "flex-start" });
-    const badgeStyles = css({ height: "1.2em", margin: "0 1.2em" });
     return (
-        <li className={cx(className, flex)} onClick={() => onClick(version)}>
+        <HistoryItemStyled type={type} onClick={() => onClick(version)}>
```
Comment (Dzuming) [inline designer/client/src/components/history/HistoryItem.tsx:34]: It's a problem with styles here:

Staging
image

locally after these changes:

image

1. Text color based on state
2. There shouldn't be a hover effect on active dot

File: designer/client/src/components/history/ProcessHistory.tsx [hunk 2]
```diff
@@ -41,13 +41,13 @@ export function ProcessHistoryComponent(props: { isReadOnly?: boolean }): JSX.El
...
         <Scrollbars
-            renderTrackVertical={(props) => <div {...props} className={styles.innerScroll} />}
+            renderTrackVertical={(props) => <TrackVertical {...props} />}
             renderTrackHorizontal={() => <div className="hide" />}
             autoHeight
             autoHeightMax={300}
             hideTracksWhenNotNeeded={true}
         >
-            <ul id="process-history">
+            <ProcessHistoryWrapper>
```
Comment (Dzuming) [inline designer/client/src/components/history/ProcessHistory.tsx:50]: Are you sure we can remove this id `id="process-history"`? maybe it's used as an anchor for some JS events (I didn't see any in the code, but still it's worth to be sure)

File: designer/client/src/components/toolbarComponents/ToolbarsLayer.tsx [hunk 3]
```diff
@@ -58,13 +58,13 @@ function ToolbarsLayer(props: { toolbars: Toolbar[]; configId: string }): JSX.El
     return (
         <DragDropContext onDragEnd={onDragEnd} onDragStart={onDragStart}>
             <SidePanel side={PanelSide.Left} className={cn(styles.left, isDragging && styles.isDraggingStarted)}>
-                <ToolbarsContainer availableToolbars={availableToolbars} side={ToolbarsSide.TopLeft} className={cn(styles.top)} />
-                <ToolbarsContainer availableToolbars={availableToolbars} side={ToolbarsSide.BottomLeft} className={cn(styles.bottom)} />
+                <ToolbarsContainer availableToolbars={availableToolbars} side={ToolbarsSide.TopLeft} />
```
Comment (Dzuming) [inline designer/client/src/components/toolbarComponents/ToolbarsLayer.tsx:61]: What was the reason for these all `className={cn(styles.bottom)}`? I didn't see any changes related to this class, but they are just removed

File: designer/client/src/components/toolbarSettings/defaultToolbarsConfig.ts [hunk 4]
```diff
@@ -1,7 +1,7 @@
 /* eslint-disable i18next/no-literal-string */
 import { ToolbarsSide } from "../../reducers/toolbars";
 import { WithId } from "../../types/common";
-import { ButtonsVariant } from "../toolbarComponents/ToolbarButtons";
+import { ButtonsVariant } from "../toolbarComponents/toolbarButtons/index";
```
Comment (Dzuming) [inline designer/client/src/components/toolbarSettings/defaultToolbarsConfig.ts:4]: Something is wrong with this import `toolbarButtons/index` it should be just `toolbarButtons`

File: designer/client/src/components/toolbarSettings/buttons/LinkButton.tsx [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import React, { ComponentType } from "react";
 import FallbackIcon from "../../../assets/img/toolbarButtons/link.svg";
 import { PlainStyleLink } from "../../../containers/plainStyleLink";
-import ToolbarButton from "../../toolbarComponents/ToolbarButton";
+import { ToolbarButton } from "../../toolbarComponents/toolbarButtons/index";
```
Comment (Dzuming) [inline designer/client/src/components/toolbarSettings/buttons/LinkButton.tsx:4]: Something is wrong with this import `toolbarButtons/index` it should be just `toolbarButtons`

File: designer/client/src/components/toolbarSettings/types.ts [hunk 6]
```diff
@@ -1,5 +1,5 @@
 import { ToolbarsSide } from "../../reducers/toolbars";
-import { ButtonsVariant } from "../toolbarComponents/ToolbarButtons";
+import { ButtonsVariant } from "../toolbarComponents/toolbarButtons/index";
```
Comment (Dzuming) [inline designer/client/src/components/toolbarSettings/types.ts:2]: Something is wrong with this import toolbarButtons/index it should be just toolbarButtons

File: designer/client/src/components/history/ProcessHistory.tsx (hunk 2 above)
Comment (bohdanprog) [reply designer/client/src/components/history/ProcessHistory.tsx:50]: I didn't find that id at any cases, hm...

File: designer/client/src/components/history/HistoryItem.tsx (hunk 1 above)
Comment (bohdanprog) [reply designer/client/src/components/history/HistoryItem.tsx:34]: fix

File: designer/client/src/components/toolbarSettings/defaultToolbarsConfig.ts (hunk 4 above)
Comment (bohdanprog) [reply designer/client/src/components/toolbarSettings/defaultToolbarsConfig.ts:4]: fix everywhere

File: designer/client/src/components/toolbarSettings/types.ts (hunk 6 above)
Comment (bohdanprog) [reply designer/client/src/components/toolbarSettings/types.ts:2]: fix everywhere

File: designer/client/src/components/history/ProcessHistory.tsx (hunk 2 above)
Comment (Dzuming) [reply designer/client/src/components/history/ProcessHistory.tsx]: Ok, so let's leave it

File: designer/client/src/components/toolbarComponents/ToolbarsLayer.tsx (hunk 3 above)
Comment (bohdanprog) [reply designer/client/src/components/toolbarComponents/ToolbarsLayer.tsx]: fix

File: designer/client/src/components/history/HistoryItem.tsx (hunk 1 above)
Comment (Dzuming) [reply designer/client/src/components/history/HistoryItem.tsx]: Nice, one more thing, there is still a hover event on active items, on staging version there is no hover effect 
image

**Review by Dzuming — APPROVED** [review]

**Pull request merged**